src/checks/noDependencyProblems/noUnusedDependencies.ts

```diff
@@ -75,9 +75,41 @@ export default async function noUnusedAndMissingDependencies() {
       `);
     }
 
-    if (missingDependenciesStdout) {
+    /** Filter out @upleveled/eslint-config-upleveled peer dependencies not listed in package.json and flagged as missing dependencies by depcheck
```

Comment:
docblock-style comments should start on the line **after** the line with the `/**`

Reply:
aaa49a7